freestyle-effects/shared/src/main/scala/effects/validation.scala

```diff
@@ -52,27 +50,31 @@ object validation {
         def fromEither[A](x: Either[E, A]): M[Either[E, A]] =
           x match {
             case Left(err) => MS.flatMap(invalid(err))((unit) => MS.pure(x))
```

Comment:
Unrelated to this PR, but we can replace `MS.flatMap(...)(_ => MS.pure(x))` with `MS.map(...)(_ => x)` which can also be written as `MS.as(...)(x)`.

Similar in `fromValidatedNel` in the `Invalid` case.